source/mesh_deformation/interface.cc

```diff
@@ -308,6 +333,18 @@ namespace aspect
 
 
 
+    template <int dim>
+    void MeshDeformationHandler<dim>::execute_initial_deformation()
+    {
+      TimerOutput::Scope timer (sim.computing_timer, "Mesh deformation initialize");
+      const AffineConstraints<double> initial_deformation_constraints = make_initial_constraints();
+
+      compute_initial_mesh_displacements(initial_deformation_constraints);
+
```

Comment:
```suggestion
```

Reply:
Why not let `compute_initial_displacements()` call `make_initial_constraints()`? This would seem more natural, and then you don't need the `execute_...()` function at all.